### DISCUSSION TITLE
Vineyard 集群模式

### DISCUSSION THREAD

**Discussion by yg000** [v6d-io/v6d#1077] [General]
您好：Vineyard 不依赖Kubernetes，可以部署集群cluster模式吗？没有找到对应文档

**Comment by sighingnow** [comment]
hi @yg000,

您好，支持的。vineyard 命令行参数有一个 `--etcd_endpoint` 参数，多个vineyardd 在启动的时候指定同一个etcd 的endpoint就可以了。

如果有后续的问题欢迎留言～
